docs/supported-tools.md

```diff
@@ -0,0 +1,271 @@
+# Supported tools
+
+The plugin supports several static analysis tools. The availability of each tool depends on the project the plugin is applied to.
+Some tools only support Java code, some only Kotlin code, and some only work on Android projects. To be precise:
+
+Tool | Java | Android<br/>(Java) | Kotlin | Android<br/>(Kotlin)
+---- | -------- | -------- | ----- | -----
+[`Checkstyle`](https://checkstyle.sourceforge.net) | :white_check_mark: | :white_check_mark: | — | —
+[`PMD`](https://pmd.github.io) | :white_check_mark: | :white_check_mark: | — | —
+[`FindBugs`](http://findbugs.sourceforge.net/) | :white_check_mark: | :white_check_mark: | — | —
+[`Detekt`](https://github.com/arturbosch/detekt) | — | — | :white_check_mark: | :white_check_mark:
+[`KtLint`\*](https://github.com/shyiko/ktlint) | — | — | ✖️ | ✖️
+[`Android Lint`\*](https://developer.android.com/studio/write/lint.html) | — | ✖️ | — | ✖️
+
+_\* Not supported [yet](https://github.com/novoda/gradle-static-analysis-plugin/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+tool%22)_
+
+For additional informations and tips on how to obtain advanced behaviours with the plugin and its tools, please refer to the
+[advanced usage](advanced-usage.md) page.
+
+## Table of contents
+ * [Enable and disable tools](#enable-and-disable-tools)
+ * [Checkstyle](#checkstyle)
+   * [Configure Checkstyle](#configure-checkstyle)
+   * [Checkstyle in mixed-language projects](#checkstyle-in-mixed-language-projects)
+ * [PMD](#pmd)
+   * [Configure PMD](#configure-pmd)
+   * [PMD in mixed-language projects](#pmd-in-mixed-language-projects)
+ * [Findbugs](#findbugs)
+   * [Configure Findbugs](#configure-findbugs)
+   * [Findbugs in mixed-language projects](#findbugs-in-mixed-language-projects)
+ * [Detekt](#detekt)
+   * [IMPORTANT: setup Detekt](#important-setup-detekt)
+   * [Configure Detekt](#configure-detekt)
+   * [Detekt in mixed-language projects](#detekt-in-mixed-language-projects)
+ * KtLint — _COMING SOON_
+ * Android Lint — _COMING SOON_
+ * [Example configurations](#example-configurations)
+
+---
+
+## Enable and disable tools
+In order to enable a tool, you just need to add it to the `staticAnalysis` closure. To enable all supported tools with their default configurations:
+
+```gradle
+staticAnalysis {
+    penalty {
+        // ... (optional)
+    }
+
+    checkstyle {}
+    pmd {}
+    findbugs {}
+    detekt {}
+}
+```
+
+To disable a tool, simply omit its closure from `staticAnalysis`. This means that, for example, this will not run any tools:
+
+```gradle
+staticAnalysis {
+    penalty {
+        // ...
+    }
+}
+```
+
+## Checkstyle
```

Comment:
@rock3r what about creating a separate md file for each tool?

Reply:
As mentioned above, was planning to do it when adding Lint. Can do in this PR if consensus is going that way.

Reply:
Continues in https://github.com/novoda/gradle-static-analysis-plugin/pull/54#discussion_r164992250